common/djangoapps/student/views.py

```diff
@@ -1238,11 +1238,20 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
                 AUDIT_LOG.warning(u"Login failed - password for {0} is invalid".format(email))
 
         # Labster changes
-        # If there was login error, suggest student to login to UK region server
-        if not request.build_absolute_uri('/')[:-1] == settings.LABSTER_REGIONAL_EDX:
-            msg = 'If you are a student in the UK, your university may now be using the UK version of Labster. Please' \
-                  ' try to log in here: <a href="{site}{login_url}">{site}</a>'
-            msg = _(msg.format(site=settings.LABSTER_REGIONAL_EDX, login_url=reverse('signin_user')))
+        # If there was login error, suggest student to login to region server
+        regions = configuration_helpers.get_value('REGIONS', settings.REGIONS)
+        current_region = request.session.get('country_code')
+        if current_region in regions.keys():
+            region_info = regions[current_region]
+            msg = _(
+                'If you are a student in the {name}, your university may now be using the {region} version of '
+                'Labster. Please try to log in here: <a href="{login_url}">{site}</a>'
+            ).format(
+                name=region_info['name'],
+                region=current_region.upper(),
+                login_url=region_info['login_url'],
+                site=region_info['url']
```

Comment:
Can we replace it with 'Please try to log in <a href="{login_url}">{region_name}</a> region' to avoid using redundant `region_info['url']`